README.md

```diff
@@ -22,7 +22,7 @@ Which will start the web application
 
 The configuration file follows a simple structure. 
 
-```json
+```js
```

Comment:
Why is this changed?

Reply:
The 3 dots symbolizing that the object/list contains more elements is not json syntax, so it gets marked red in markdown. It is however javascript syntax.